PublicationsRouter/v2/api/MatchingConfig.md

```diff
@@ -0,0 +1,42 @@
+# Matching Config Structure
+Document contains structure of matching configuration JSON for both submission and retrieval. Used in POST/GET requests respectively against endpoint `/config`.
+
+## JSON Submitted Structure
+The structure of a JSON submission for matching configuration must match the following format: 
+```
+{
+  "name_variants": ["list of name variants the institution is known by"],
+  "postcodes" : ["list of postcodes where authors may list their affiliation address"],
+  "domains": ["list of domain names the institution owns or operates under"],
+  "grants": ["list of grant numbers affiliated with the institution"],
+  "author_ids": [ // list of identifier objects of format
+		{
+			'type': 'the type of identifier, either email or ORCID', 
+			'id': 'the value, so either the email address or the ORCID'
+		}
+	]
+}
+```
```

Comment:
Would be good to provide a concise example, or if you move the GET above the POST, you can simply refer reader to the GET to see examples of the data to be submitted for each key.